man/tippecanoe.1

```diff
@@ -29,7 +29,11 @@ significant travel corridor.
 .PP
 The easiest way to install tippecanoe on OSX is with Homebrew \[la]http://brew.sh/\[ra]:
 .PP
+.RS
```

Comment:
What does this notation mean?

Reply:
This is `nroff -man` format, the ancient markup language of manpages. I haven't written one by hand for so long that I can't remember what `.RS` does (this one is generated by `md2man-roff`), although I used to care an awful lot about nroff and troff, but:

```
$ man 7 man
MAN(7)                 Miscellaneous Information Manual                 MAN(7)

NAME
     man – legacy formatting language for manual pages
…
     RS, RE    reset the left margin: [width]
```